package.json

```diff
@@ -2,11 +2,15 @@
   "name": "Duckling",
   "version": "0.0.1",
   "scripts": {
-    "start": "gulp && tsc && concurrently \"gulp watch\" \"npm run tsc:w\" \"npm run electron\" ",
+    "start": "npm run build && concurrently \"npm run watch\" \"npm run duckling\" ",
+    "test": "tsc && jasmine",
+    "build": "gulp && tsc",
+    "watch": "concurrently \"gulp watch\" \"npm run tsc:w\"",
     "tsc": "tsc",
     "tsc:w": "tsc -w",
     "typings": "typings",
-    "electron": "electron build/electron/background-main.js",
+    "duckling": "electron build/electron/background-main.js",
+    "duckling-test": "electron build/duckling-tests/background-test-main.js",
```

Comment:
An electron test shell is no longer planned for the jasmine tests, you should remove the remnants of it here.